CHANGES.txt

```diff
@@ -1,3 +1,12 @@
+New in master
+=============
+
+Other Information
```

Comment:
Please remove the changelog entry. Users don’t really care about links being made https, nor should they.

Reply:
Sure. I didn't include the changelog entry initially but updated the PR after the first submission because of the instructions indicating a changelog entry was required. I'll revert this change.